src/app/tokens.scss

```diff
@@ -171,9 +171,9 @@ $border-focus-width: 3px;
 
 // Drop shadow
 $box-shadow-xs:
-  0 4px 6px 1px rgba(29, 17, 51, 0.04),
-  0 1px 8px 1px rgba(9, 32, 77, 0.12),
-  0 3px 5px -1px rgba(29, 17, 51, 0.12);
+  0 4px 6px rgba(29, 17, 51, 0.02),
+  0 2px 8px rgba(9, 32, 77, 0.08),
+  0 3px 5px -2px rgba(29, 17, 51, 0.08);
```

Comment:
@codemist I’ve copied your value for `$box-shadow-xs` over from your open PR [here](https://github.com/mozilla/blurts-server/pull/5287/files#diff-264eb69f4bc4b07a3eb48ca05441a751ee86a679cb867a6b91b40193feb10bd3R173-R176).